### PR TITLE
Open 2020 with one

### PR DESCRIPTION
**Pull request by agrayver**
It is in press, but can already be retrieved.

### CODE REVIEW

File: publications-2020.bib [hunk 1]
```diff
@@ -0,0 +1,15 @@
+% Encoding: US-ASCII
+
+@article{10.1093/gji/ggaa039,
+    author = {Käufl, J S and Grayver, A V and Comeau, M J and Kuvshinov, A V and Becken, M and Kamm, J and Batmagnai, E and Demberel, S},
```
Comment (bangerth) [inline publications-2020.bib]: It's unhappy with the umlaut. Can you write it as `\"a`?

Comment (agrayver) [reply publications-2020.bib]: Fixed

**Comment by bangerth** [comment]
Thanks!

**Pull request merged**